dm/worker/relay.go

```diff
@@ -308,13 +308,20 @@ func (h *realRelayHolder) setResult(result *pb.ProcessResult) {
 }
 
 // Result returns the result of the relay
+// Note this method will omit the `Error` field in `pb.ProcessError`
```

Comment:
why do we need to omit the `Error` field in `pb.ProcessError` (and `Subtask.Result` too)?